src/Reflection/ReflectionClass.php

```diff
@@ -101,7 +101,7 @@ class ReflectionClass implements Reflection
     private int $endColumn;
 
     /** @var class-string|null */
-    private string|null $parentClassName;
+    private string|null $parentClassName = null;
```

Comment:
This is always initialized in the constructor: shouldn't need a default value.

Reply:
I added it, because the tests failed with "must not be accessed before initialization"

https://github.com/Roave/BetterReflection/actions/runs/5713496659/job/15479000753